playgrounds/nextjs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "faker-nextjs-test",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
```

Comment:
Could you please add a `compile` alias? This will then be executed after the tests via GitHub CI.
See root package.json line 10 (in your branch)

Reply:
Added! I also updated the next.js example page to import and use faker so that it will compile with faker usage.